smart-contracts/contracts/PublicLock.sol

```diff
@@ -26,6 +26,12 @@ contract PublicLock is ILockCore, ERC165, IERC721, Ownable {
     bytes data; // Note: This can be expensive?
   }
 
+  // Events
+  event PriceChanged(
```

Comment:
Looks good. I typically use a pattern of logging both the old and new values in events, so `uint indexed oldKeyPrice, uint indexed newKeyPrice`. It's not strictly necessary, but it can help to achieve the ability to "reconstruct" the application state from the event logs only.

Reply:
That's a good practice, I'll update the PR. Thanks!